art/attacks/hop_skip_jump.py

```diff
@@ -63,6 +65,7 @@ def __init__(self, classifier, targeted=False, norm=2, max_iter=50, max_eval=100
                   'init_eval': init_eval,
                   'init_size': init_size,
                   'curr_iter': 0,
+                  'batch_size': 1
```

Comment:
```suggestion
                  'batch_size': batch_size
```

Reply:
This attack currently does not support batching.